chapter11/drjt/ex1.pm

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/perl
+use 5.010;
+use strict;
+use warnings;
+
+package ex1;
```

Comment:
You probably don't want to give this an exercise-specific name, just as we'll be updating it in later chapters.